README.md

```diff
@@ -44,7 +44,7 @@ $database = InfluxDB\Client::fromDSN(sprintf('influxdb://user:pass@%s:%s/%s', $h
 $client = $database->getClient();
 ```
 
-### Reading
+### Reading data
```

Comment:
This change is just to be more consistent with the terms below where it says 'Writing data' not just 'Writing'